### DISCUSSION TITLE
v0.0.0-alpha

### DISCUSSION THREAD

**Discussion by jake-baxter** [jake-baxter/rblx-train-driving-system#3] [General]
**THIS IS A TEST RELEASE SO THIS IS NOT FINAL**

# **_v0.0.0-alpha_**

Any issues? Check the readme.md file for how to report it.

# Check [this out](https://www.roblox.com/library/8026850378/v0-0-0-alpha-train-driving-system) if you want to download off roblox

## What's Changed
* Create TrainModule.lua by @jake-baxter in https://github.com/jake-baxter/rblx-train-driving-system/pull/1
* v0.0.0-alpha pull by @jake-baxter in https://github.com/jake-baxter/rblx-train-driving-system/pull/2

## New Contributors
* @jake-baxter made their first contribution in https://github.com/jake-baxter/rblx-train-driving-system/pull/1

**Full Changelog**: https://github.com/jake-baxter/rblx-train-driving-system/commits/v0.0.0-alpha

# The files

`v0.0.0-alpha-zip.zip`  -  Zip file with source code

`TrainModuleServerStorage.rbxm` - Importable Roblox ModuleScript file for the main train module

`TrainModuleServerStorage.rbxm` - Importable Roblox ScreenGUI object file for the main UI  in the system

`TrainModuleExample.rbxm` - Downloadable example model

`TMExamplePlace.rbxl` - Downloadable example place

<hr /><em>This discussion was created from the release <a href='https://github.com/jake-baxter/rblx-train-driving-system/releases/tag/v0.0.0-alpha'>v0.0.0-alpha</a>.</em>

**Comment by marcusleonas** [comment]
I was wondering.... Will a GUI be implemented?

**Reply by jake-baxter** [reply]
https://github.com/jake-baxter/rblx-train-driving-system/issues/9

**Comment by Sindiously** [comment]
Will you be updating this anytime soon?
